src/core/server/plugins/plugin_context.ts

```diff
@@ -206,6 +206,7 @@ export function createPluginSetupContext<TPlugin, TPluginDependencies>(
       getImportExportObjectLimit: deps.savedObjects.getImportExportObjectLimit,
       setRepositoryFactoryProvider: deps.savedObjects.setRepositoryFactoryProvider,
       setStatus: deps.savedObjects.setStatus,
+      addExtendedSavedObjectsMappings: deps.savedObjects.addExtendedSavedObjectsMappings,
```

Comment:
Shall we rename to `setExtendedSavedObjectsMappings`? `add` may confused. The old extended saved objects mappings will be override.

Reply:
I've modified the variable `extendedSavedObjectsMappings` to array type, it meant to make all other plugins can add their extended mappings, so `addExtendedSavedObjectsMappings` makes more sense?